users/api/serializers.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework.serializers import ModelSerializer
+
+from users.models import Profile
+
+
+class ProfileDetailSerializer(ModelSerializer):
+    class Meta:
+        model = Profile
+        fields = ['user', 'image', 'bio', 'year', 'branch']
```

Comment:
The `user` field in the `json` that's received should be another object having the details of user like _first_name_, _last_name_ and _email_id_.

Reply:
Done